src/views/Accessibility/AccessibilityRequest/Create/index.tsx

```diff
@@ -160,13 +151,16 @@ const Create = () => {
                         <Label htmlFor="508Request-IntakeId">
                           {t('newRequestForm.fields.project.label')}
                         </Label>
+                        <HelpText id="508Request-IntakeId-HelpText">
+                          {t('newRequestForm.fields.project.helpText')}
+                        </HelpText>
                         <FieldErrorMsg>{flatErrors.intakeId}</FieldErrorMsg>
                         <ComboBox
                           id="508Request-IntakeComboBox"
                           name="intakeComboBox"
                           inputProps={{
                             id: '508Request-IntakeId',
-                            name: 'intakeId'
```

Comment:
Is there a reason why this is getting deleted?

Reply:
I think it was honestly an accident. Good catch, Chris!